examples/3D_contour.py

```diff
@@ -0,0 +1,165 @@
+"""
+A comprehensive example that shows the whole workflow of
+1) Loading data
+2) Defining the model structure for a joint distribution
+3) Estimating the parameter values of the model ("Fitting")
+4) Computing a 3D environmental contour
+
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from skimage.measure import marching_cubes
+import pandas as pd
+
+from virocon import (
+    read_ec_benchmark_dataset,
+    GlobalHierarchicalModel,
+    LogNormalDistribution,
+    ExponentiatedWeibullDistribution,
+    DependenceFunction,
+    WidthOfIntervalSlicer,
+    HighestDensityContour,
+    plot_marginal_quantiles,
+    plot_dependence_functions,
+)
+
+
+# Load sea state measurements. 
+
```

Comment:
Delete this empty line. Comments that describe a block of code should be right on top of it.

By the way.. did you run the formatter black on the script?